lib/mongo/index/view.rb

```diff
@@ -149,21 +154,59 @@ def create_one(keys, options = {})
       #
       # @since 2.0.0
       def create_many(*models)
```

Comment:
Wouldn't the following work?

`def create_many(*models, **options)`

Reply:
I believe that creates breaking changes in the API. Consider the following example:

`create_many({ key: { random: 1 } })`

When the method definition is `def create_many(*models)`, the `models` argument is `[{key: { random: 1 } }]`.

However, when the method definition is `def create_many(*models, **options)`, the `models` argument is an empty array, and `options` contains the specified Hash.

(Also happy to chat about this tomorrow.)

Reply:
A simplified example for communication purposes:

```
def func1(*models)
  puts "MODELS: #{models}"
end

def func2(*models, **options)
   puts "MODELS: #{models}"
   puts "OPTIONS: #{options}"
 end

func1({ key: 'hello'})
# MODELS: [{:key=>"hello"}]

func2({ key: 'hello' })
# MODELS: []
# OPTIONS: {:key=>"hello"}
```